compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -468,6 +468,11 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         no_core, CrateLevel, template!(Word), WarnFollowing,
         @only_local: true, experimental!(no_core)
     ),
+    gated!(
+        rustc_ub_check, CrateLevel, template!(Word), WarnFollowing,
+        @only_local: true,
+        "`#![rustc_ub_check]` is an internal feature of the standard library, mark don't optimize UB Checks",
```

Comment:
why should Mark not optimize UB checks?

Reply:
See https://github.com/rust-lang/rust/issues/120539. In previous versions, the standard library debug assert did not work.

Reply:
Thanks! :3